packages/client/src/features/routing/glsp-polyline-edge-router.ts

```diff
@@ -0,0 +1,32 @@
+/********************************************************************************
+ * Copyright (c) 2024 EclipseSource and others.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * This Source Code may also be made available under the following Secondary
+ * Licenses when the conditions for such availability set forth in the Eclipse
+ * Public License v. 2.0 are satisfied: GNU General Public License, version 2
+ * with the GNU Classpath Exception which is available at
+ * https://www.gnu.org/software/classpath/license.html.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+ ********************************************************************************/
+import { GConnectableElement, GParentElement, GRoutableElement, Point, PolylineEdgeRouter } from '@eclipse-glsp/sprotty';
+import { injectable } from 'inversify';
+
+@injectable()
+export class GLSPPolylineEdgeRouter extends PolylineEdgeRouter {
```

Comment:
If I recall correctly we decided in a previous discussion regarding the naming of the feature modules to not use the GLSP prefix if possible and rather rexport the parent sprotty class with a sprotty prefix.

So we should probably change this to 
```suggestion
export class PolylineEdgeRouter extends SprottyPolylineEdgeRouter {
```
and add the corresponding rexport to the glsp-sprotty package

Reply:
Good point, I pushed an update that properly deals with the sprotty-fixed versions and the re-export!